spec/02-integration/09-hybrid_mode/10-forward-proxy_spec.lua

```diff
@@ -166,6 +170,15 @@ for _, strategy in helpers.each_strategy() do
           if auth_on then
             assert.matches("accepted basic proxy%-authorization", contents)
           end
+
+          -- check the debug log of the `cluster_use_proxy` option
+          path = pl_path.join("servroot2", "logs", "error.log")
```

Comment:
It could be `assert.logfile("servroot2/logs/error.log").has.line("...", true)`

Reply:
let me try